### ISSUE TITLE
Publish 2021.1 release to Github?

### ISSUE THREAD

**Issue by takanuva15** [weisJ/auto-dark-mode#35]
Hi, can you please published the 211.* version of this plugin to GitHub for direct download? I only see the 2020.3 version currently

**Comment by weisJ** [comment]
All new versions should be available through the plugin settings by now. Nevertheless I can upload a build to GitHub just in case.

**Comment by takanuva15** [comment]
What do you mean by plugin settings? (I checked the settings and there doesn't seem to be any upgrade button?)

![image](https://user-images.githubusercontent.com/6986426/114479773-29fe4f80-9bcf-11eb-80d0-67ed2e0cb220.png)

**Comment by weisJ** [comment]
The "Plugins" tab right below "Editor". Though you seem to have installed the plugin so what is the problem exactly?

**Comment by takanuva15** [comment]
Ok I see; you were talking about automatic updates that are uploaded to the JetBrains marketplace and then the IDE would prompt you to update the plugin. I had already checked the Marketplace and it wasn't available, so that's why I was asking for the plugin to be manually published.

I actually tried cloning the repo and compiling the plugin myself to make it work with 211.* since there was no release published yet. That actually worked for me somehow so that's how I was able to take the screenshots above with 2021.1 lol
